frontend/app/utils/string-utils.ts

```diff
@@ -59,6 +59,15 @@ export function removeInvalidInputCharacters(value: string) {
   return value.replace(invalidInputCharactersRegex, '');
 }
 
+/**
+ *
+ * @param value - The input string to be checked.
+ * @returns - Returns true if the input does not contain digits, otherwise false.
+ */
+export function doesNotContainsDigits(value: string) {
```

Comment:
Change it to `hasDigits` and it returns `true` if it contains Digits and `false` if it doesn't. Use negation when needed `!hasDigits(str)`.

Reply:
You can create another method that use  `hasDigits` if you doesn't want to negate the return value with zod refine mehod.

```ts
export function hasNoDigits(value: string) {
 return !hasDigits(value);
}
```

Reply:
It's generally recommended to use affirmative names for boolean methods and variables, rather than using negation. This can help improve code readability and reduce cognitive load. For example, instead of using `!hasNoDigits`, you could use `hasDigits` and negate the result if needed.

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig